angular2/src/angular2TextMask.ts

```diff
@@ -33,12 +33,10 @@ export class MaskedInputDirective implements OnInit, AfterViewInit, ControlValue
   private _onTouched = () => {}
   private _onChange = (_: any) => {}
 
-  private isInit: boolean = false
-
   constructor(private renderer: Renderer, private element: ElementRef) {}
 
   ngAfterViewInit() {
-    !this.isInit && this.setupMask()
+    !this.textMaskInputElement && this.setupMask()
```

Comment:
I sort of agree with @PerfectPixel but this is a stylistic change, so I don't think this should be a blocker to this PR